ibazel/fs_watcher/fs_watcher.go

```diff
@@ -12,36 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package fs_watcher
 
 import (
 	"github.com/fsnotify/fsnotify"
 )
 
-type SourceEventHandler struct {
-	SourceFileEvents  chan fsnotify.Event
-	SourceFileWatcher *fsnotify.Watcher
-}
-
-func (s *SourceEventHandler) Listen() {
```

Comment:
This is a bad diff; this file was deleted not renamed. It is not needed since now individual files are not watched, rather just their directories